operator/operator.make

```diff
@@ -1,3 +1,5 @@
+export SHELLOPTS:=$(if $(SHELLOPTS),$(SHELLOPTS):)pipefail:errexit
```

Comment:
This affects all commands invoked by make. Can we avoid this? It might be surprising.